avm/ptn/app/paas-ase-cosmosdb-tier4/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,63 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```

Comment:
Incorrect description for waf-aligned. Please copy the correct name & description for literally any other waf-test of any other module 😄

Reply:
updated